docs/man/docker-stats.1.md

```diff
@@ -16,6 +17,9 @@ Display a live stream of one or more containers' resource usage statistics
 Note: this functionality currently only works when using the *libcontainer* exec-driver.
 
 # OPTIONS
+**-d**, **--display**=-1
+  Show only count displays, then exit
```

Comment:
I'm sorry, I don't know what `count displays` are - is there some better wording for this? (or explain it to us and we'll figure out something :)